addons/website_event/data/website_snippet_data.xml

```diff
@@ -6,7 +6,10 @@
         <field name="name">Upcoming Events</field>
```

Comment:
Commit message typos:

- The access rights and published state still works as before -> work